src/ui/views/ViewMobileOnboarding.qml

```diff
@@ -256,18 +256,27 @@ VPNFlickable {
             Layout.minimumHeight: VPNTheme.theme.windowMargin
         }
 
-        Column {
+        ColumnLayout {
             id: panelBottomContent
             Layout.preferredWidth: parent.width
             spacing: VPNTheme.theme.windowMargin
 
+            VPNInterLabel {
+                Layout.maximumWidth: parent.width
+                Layout.alignment: Qt.AlignCenter
+
+                text: VPNl18n.FreeTrialsStartYourFreeTrial
+                color: VPNTheme.colors.white
+            }
```

Comment:
From the [Figma specs](https://www.figma.com/file/kpz1dqP2terBMIZLjiOhev/Free-Trials?node-id=109%3A289) it seems like the font style should supposed to be _Inter SemiBold 600_ and the color not plain white.

Reply:
Correct, we don't use Inter Semibold 600 or white.80 (#FFFFFFCC) anywhere else - will double check with @peterbenvenuto